jest-puppeteer.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  browserContext: 'incognito'
```

Comment:
Why?

Reply:
This will be removed, as I'll revert to the newly introduced `playwright` integration.